webpack/index.js

```diff
@@ -36,11 +39,31 @@ const getClientConfig = () => {
   }
 }
 
+const getNovoClientConfig = () => {
+  switch (true) {
+    case env.isDevelopment:
+      console.log("[Force Novo] Building client-side development code...")
+      return clientNovoDevelopmentConfig
```

Comment:
Something like: 
```jsx
return merge.smart(clientNovoDevelopmentConfig, clientCommonConfig)
```

Reply:
See the previous comment about duplication.